gateway/sensor.go

```diff
@@ -95,6 +94,11 @@ type Gateway struct {
 	rstPin   string
 }
 
+// global variable to track if the captureOutputToLogs goroutine has started
+// If the build errors and needs to build again, we only want to start the logging goroutine
+// once.
+var logsStarted bool = false
```

Comment:
if we have multiple sensors and close one is closed this will be overwritten. we might want it to be contained in the sensors struct or mapped to a sensor name.

Reply:
Okay yeah I see i didn't think about having multiple thanks for pointing that out, I can't put it in the gateway struct because it gets overwritten on rebuild, I think making a map with the gateway name would work, will try to do that.